cmake/modules/AddSwift.cmake

```diff
@@ -1603,6 +1618,7 @@ function(add_swift_library name)
 
     _add_swift_library_single(
       ${name}
+      ""
```

Comment:
This is why I would suggest adding a named argument /assuming/ that is the correct thing to do here.

Then it would be clear that this is not a bug since you would have:

MODULE_TARGET ""

That being said, if you do what I suggested above, then this argument could be removed.